examples/fabric-admin/device_manager/DeviceManager.cpp

```diff
@@ -141,11 +140,15 @@ void DeviceManager::OpenRemoteDeviceCommissioningWindow(EndpointId remoteEndpoin
     uint16_t discriminator =
```

Comment:
nit, can you remove ` StringBuilder<kMaxCommandSize> commandBuilder;` from above on line 137